specification/cognitiveservices/Language.Conversations/client.tsp

```diff
@@ -188,10 +186,6 @@ namespace Language.ConversationsClientCustomizations;
   "ActionContent",
   "csharp"
 );
-@@clientName(CustomConversationSummarizationTaskParameters,
```

Comment:
It's already defined in
https://github.com/Azure/azure-rest-api-specs/blob/1e75340397ab8326c13e8c2eeb2d4b65be1d0eba/specification/cognitiveservices/Language.Conversations/client.tsp#L84-L87